README.md

```diff
@@ -64,6 +64,7 @@ Some tests support these environment variables (work in progress):
   - SOF_ALSA_OPTS contains optional parameters passed on both play and record.
   - SOF_APLAY_OPTS and SOF_ARECORD_OPTS contain optional parameters passed additionally on play and record respectively.
 These options are applied to the selected tool (alsa or tinyalsa) based on the value of SOF_ALSA_TOOL 
+  = TEST_WITH_PIPEWIRE (default: false) allows to use pipewire instead of ALSA direct mode. Currently only works with check-performance.sh.
```

Comment:
```suggestion
  = SOF_TEST_PIPEWIRE (default: false) allows to use pipewire instead of ALSA direct mode. Currently only works with check-performance.sh.
```
Use a prefix / "namespace"